konfera/templates/konfera/event_speakers.html

```diff
@@ -0,0 +1,17 @@
+{% extends 'konfera/base.html' %}
+
+{% block title %}<title>{{ event }} Speakers</title>{% endblock %}
+
+
+{% block content %}
+        <div>
+            {% for talk in talks %}
+                <h4>Talk: {{ talk.title }}</h4>
+               <p>Speaker#1 : <a href="{{ talk.primary_speaker.social_url }}">{{ talk.primary_speaker.title }} {{ talk.primary_speaker.first_name }} {{ talk.primary_speaker.last_name }}</a></p>
```

Comment:
We override the `__str__` in `Speaker` model so you can just simple write `{{ talk.primary_speaker }}` instead of:

        {{ talk.primary_speaker.title }} {{ talk.primary_speaker.first_name }} {{ talk.primary_speaker.last_name }}